tools/run_tests/xds_k8s_test_driver/bin/cleanup.sh

```diff
@@ -50,10 +50,10 @@ cd "${XDS_K8S_DRIVER_DIR}"
 if [[ "$1" == "--nosecure" ]]; then
   shift
   ./run.sh bin/run_td_setup.py --cmd=cleanup "$@" && \
-  ./run.sh bin/run_test_client.py --cmd=cleanup "$@" && \
+  ./run.sh bin/run_test_client.py --cmd=cleanup --cleanup_namespace "$@" && \
```

Comment:
Missed this when updated the framework to use separate namespaces for the client and the server.